tests/MercadoPago/Resources/Mocks/Response/Order/order_capture.json

```
@@ -0,0 +1,36 @@
+{
+  "id": "01HRYFWNYRE1MR1E60MW3X0T2P",
+  "type": "online",
+  "total_amount": "1000.00",
+  "external_reference": "ext_ref_1234",
+  "site_id": "MLB",
+  "created_date": "2024-09-02T22:04:01.880469Z",
+  "last_updated_date": "2024-09-02T22:04:04.429289Z",
+  "status": "processed",
+  "status_detail": "accredited",
+  "type_config": {
+    "capture_mode": "manual"
+  },
+  "transactions": {
+    "payments": [
+      {
+        "id": "01HRYFXQ53Q3JPEC48MYWMR0TE",
+        "reference_id": "123456789",
+        "status": "processed",
+        "status_detail": "accredited",
+        "amount": "1000.00",
+        "payment_method": {
+          "id": "master",
+          "type": "credit_card",
+          "token": "card_token",
+          "installments": 1
+        }
+      }
+    ]
+  },
+  "payer": {
+    "email": "test_1731350184@testuser.com"
+  },
+  "processing_mode": "automatic",
+  "marketplace": "NONE"
+}
```

De acordo com o [Swagger](https://web.furycloud.io/mp-public-order-api/specs-hub/spec/5d077b6f-61b2-4b3a-8333-7a64ee547448), a resposta do endpoint de Capture é simplificada, não precisamos mocar todos os campos.
```suggestion
{
  "id": "01HRYFWNYRE1MR1E60MW3X0T2P",
  "status": "processed",
  "transactions": {
    "payments": [
      {
        "id": "01HRYFXQ53Q3JPEC48MYWMR0TE",
        "reference_id": "123456789",
        "status": "processed",
        "amount": "1000.00"
      }
    ]
  }
}
```